TicketSwap/comets
# Update packages & use TypeScript

## Description
I've started looking for TSDX alternatives (to improve the bundle size) so comets is the perfect project for this. I've started with updating all the packages and already using some plugins like Terser to reduce the bundle size. The next step is to convert this project to TypeScript. If that is a succes we can look into improving Solar as well.

Critical note: Somehow I don't get the export of type definitions to work but we don't need that for this project. If we want to move solar to the same bundling strategy we should investigate this.

## How to test
* Run the build
* Copy the dist folder to a local project and verify the icons are still working as expected 👍 

## Screenshots
New
<img width="1914" alt="new-bundle" src="https://user-images.githubusercontent.com/14276144/112143363-0f404880-8bd8-11eb-91c6-60fce022104a.png">

After adding typescript
<img width="957" alt="" src="https://user-images.githubusercontent.com/14276144/112299947-99e87c80-8c98-11eb-8d60-4cdbdc77b340.png">


Old
<img width="1911" alt="old-bundle" src="https://user-images.githubusercontent.com/14276144/112143370-10717580-8bd8-11eb-8b4d-4c81d95ed54d.png">
